gaia-fast-list.js

```diff
@@ -781,11 +807,21 @@ Internal.prototype = {
       return this.populateItem(replacement, i);
     }
 
-    el.style.borderTop = !record.gfl_isFirst
-      ? 'solid 1px var(--border-color, #e7e7e7)'
-      : '0';
+    el.classList.toggle('first', !!record[keys.first]);
```

Comment:
Using a class instead to prevent first-child styling being wiped when we remove the large `style` attr before caching